docs/guides/define_agent.md

```diff
@@ -0,0 +1,202 @@
+            version: ==1.31.0
+          open-aea-test-autonomy:
+            version: ==0.10.0.post1
+        default_connection: null
+        ---
+        public_id: valory/hello_world_abci:0.1.0
+        type: skill
+        models:
+          benchmark_tool:
+            args:
+              log_dir: ${str:/benchmarks}
+          params:
+            args:
+              hello_world_message: ${str:HELLO_WORLD!}
+              service_registry_address: ${str:null}
+              share_tm_config_on_startup: ${bool:false}
+              on_chain_service_id: ${int:null}
+              setup:
+                all_participants: ${list:[]}
+                safe_contract_address: ${str:'0x0000000000000000000000000000000000000000'}
+                consensus_threshold: ${int:null}
+              tendermint_url: ${str:http://localhost:26657}
+              tendermint_com_url: ${str:http://localhost:8080}
+        ---
+        public_id: valory/abci:0.1.0
+        type: connection
+        config:
+          target_skill_id: valory/hello_world_abci:0.1.0
+          host: ${str:localhost}
+          port: ${int:26658}
+          use_tendermint: ${bool:false}
+        ---
+        public_id: valory/ledger:0.19.0
+        type: connection
+        config:
+          ledger_apis:
+            ethereum:
+              address: ${str:http://localhost:8545}
+              chain_id: ${int:31337}
+              poa_chain: ${bool:false}
+              default_gas_price_strategy: ${str:eip1559}
+        ---
+        public_id: valory/p2p_libp2p_client:0.1.0
+        type: connection
+        config:
+          nodes:
+          - uri: ${str:acn.staging.autonolas.tech:9005}
+            public_key: ${str:02d3a830c9d6ea1ae91936951430dee11f4662f33118b02190693be835359a9d77}
+          - uri: ${str:acn.staging.autonolas.tech:9006}
+            public_key: ${str:02e741c62d706e1dcf6986bf37fa74b98681bc32669623ac9ee6ff72488d4f59e8}
+        cert_requests:
+        - identifier: acn
+          ledger_id: ethereum
+          message_format: '{public_key}'
+          not_after: '2023-01-01'
+          not_before: '2022-01-01'
+          public_key: ${str:02d3a830c9d6ea1ae91936951430dee11f4662f33118b02190693be835359a9d77}
+          save_path: .certs/acn_cosmos_9005.txt
+        - identifier: acn
+          ledger_id: ethereum
+          message_format: '{public_key}'
+          not_after: '2023-01-01'
+          not_before: '2022-01-01'
+          public_key: ${str:02e741c62d706e1dcf6986bf37fa74b98681bc32669623ac9ee6ff72488d4f59e8}
+          save_path: .certs/acn_cosmos_9006.txt
+        is_abstract: true
+        ```
+
+3. **Create an entry for your agent in the local registry.** Add the corresponding entry to the local registry index file (`./packages/packages.json`). You must add the entry to the `dev` section, because it is a component being developed by you. You can use a placeholder for its hash value, as it will be corrected afterwards:
```

Comment:
This should be done by the `aea packages lock` command, please open an issue on the open-aea repo

Reply:
Ok. But for now we don't have any other way of doing it, right?

Reply:
https://github.com/valory-xyz/open-aea/issues/609